README.md

```diff
@@ -52,6 +63,12 @@ To use NonDex, add the plugin to the plugins section under the build section in
 </project>
 ```
 
+Before running Nondex, run:
+    
+    mvn install -DskipTests
```

Comment:
While it's probably good practice that people compile and run tests normally before using NonDex, technically it's not necessary as NonDex will compile the project if not already compiled.